docs_src/examples/Ch-ExamplesSendingAndConsumingBinary.md

```diff
@@ -185,3 +185,49 @@ data inside the EdgeX Reading's `BinaryValue` field.
 Using this approach, an Event can be sent containing data containing an
 arbitrary, flexible structure. Use cases could be a Reading containing
 multiple images, a variable length list of integer read-outs, etc.
+
+### Creating A CBOR Payload For Use With PUT Commands
+
+To create a CBOR payload that can, for example, be used with `PUT` commands,
+we first need to create a CBOR decoder (from a given set of CBOR bytes),
+along with a CBOR code handle. We` decode into an EdgeX event where, as
+far as the codec is concerned, the Go library recommended for parsing
+into an EdgeX event is https://github.com/ugorji/go
+
+What follows below is the `golang` code that accomplishes the steps above:
+
+``` golang
+package main
+
+import (
+  “io/ioutil”
+
+  “github.com/ugorji/go/codec”
+)
+
+func main() {
+    // Set up some CBOR data.
+	result := make(map[string]string)
+	result["EnableRandomization_Binary"] = "true"
+
+	// Create a CBOR decoder from the CBOR bytes, along with a CBOR code handle
+	var h codec.Handle = new(codec.CborHandle)
+	var input = make([]byte, 0)
+	var dec *codec.Encoder = codec.NewEncoderBytes(&input, h)
+
+	// Decode into an EdgeX Event
+	err := dec.Encode(result)
+	check(err)
+
+	// Fetch binary data and write to a file
+	ioutil.WriteFile("/Users/johnpoe/Desktop/Decoded_Binary", input, 0644)
```

Comment:
Can we create a constant at the top of this "file" which contains the location of the file? That way users that are not familiar with Go can simply edit the contents to get the desired results.

Reply:
Mixed on this point--I think that "documentation" code like this should be written to be as readable as possible, not as maintainable as possible. Hardcoding a filename means a reader won't need to refer up the file to doublecheck that it's a path, etc.

I don't feel strongly here, though, either way is fine.

Reply:
I agree. You are right, this is for documentation purposes and should be treated a little differently.